src/main/java/lotto/domain/LottoGame.java

```diff
@@ -19,15 +20,22 @@ public LottoGame(Lottos lottos, Money money) {
         this.money = money;
     }
 
-    public List<Lotto> lottos() {
-        return lottos.values();
-    }
-
-    public LottoResult check(Lotto winningLotto) {
+    public LottoResult findWinner(WinningLotto winningLotto) {
         return lottos.findResult(winningLotto);
     }
 
     public double rateOfReturn(LottoResult result) {
         return money.rateOfReturn(result.getTotal());
     }
+
+    public int lottoCount() {
+        return lottos.values().size();
```

Comment:
```suggestion
        return lottos.size();
```
위임하는 코드만 존재해도 괜찮음
OOP에서 흔히 발생하는 코드임